### PR TITLE
Outline view

### PR DESCRIPTION
**Pull request by cbovino**
Co-Authored by @alexbednarek4 

tldr: We built a new view in VSCode to outline the structure of a graph database.

We utilize the graphoutline function built by our team members to grab all the data pertaining to the graph database structure. We then created a data provider which we provided the correct structure of the data to the view.

Package.json- We properly structured the view and command to execute the tree data provider

extension.ts - We brought in our outline provider. We created an new instance of the outline provider, and registered it along with the command required to build the view.

outlineProvider.ts - We created a treedata provider class where we parsed through all the data regarding the database. In our class constructor, we utilize the graphOutline function to receive the data about the database. We then used the provided structure from the VSCode api to create our tree items in our view. We export the class to be used in other files

### CODE REVIEW

**Review by alexdrew — APPROVED** [review]
Code Looks good:

Future Updates:
- Change execute command from Hello World
- Review uni/bi outline

- [ ]
